src/Rules/RuleInfoExtensions.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SonarLint.VisualStudio.Rules
+{
+    public static class RuleInfoExtensions
```

Comment:
Why create a new class instead of using RuleExtensions class?

Reply:
Should have been named RuleInfoExtensions and not RuleExtensions. That's why I didn't see there was a class for that already :-P

Reply:
it's not just about RuleInfo that's why it was not RuleInfoExtensions. I wrote extensions for 3 different classes and I didn't want to create a class for each extension so I have aggregated them in RuleExtensions.